custom_components/econet300/binary_sensor.py

```diff
@@ -84,7 +85,9 @@ def create_binary_entity_description(key: str) -> EconetBinarySensorEntityDescri
     entity_description = EconetBinarySensorEntityDescription(
         key=key,
         translation_key=camel_to_snake(key),
-        device_class=ENTITY_BINARY_DEVICE_CLASS_MAP.get(key, None),
+        device_class=ENTITY_BINARY_DEVICE_CLASS_MAP.get(
```

Comment:
Such functions can be moved too common_functions and called from there

Code would become cleaner and easier to read

for example function could be called `device_class=getBinaryDeviceClass(key)`